website/content/docs/concepts/client-count.mdx

```diff
@@ -188,6 +193,33 @@ To reduce the number of non-entity tokens in use, consider switching to an authe
 [AppRole](/docs/auth/approle) instead of handing out directly-created tokens. Ensure that entities and
 [entity aliases](/api/secret/identity/entity-alias) exist for all login methods used to create batch tokens.
 
+### Tracking non-entity tokens
+
+As of Vault 1.9, non-entity tokens are tracked as unique clients based on the policies the token
+has access to and the namespace in which it was created. Non-entity tokens that are assigned 
+the same set of policies and are created in the same namespace will be tracked as the same client. 
+Conversely, if two non-entity tokens have a different policy set or are created in different namespaces
+they will be tracked as two separate clients. 
+
+Please note that before the release of Vault 1.9, non-entity tokens were each tracked separately. That is 
+to say, two non-entity tokens would always be counted as two separate clients.
+
+## Auditing clients
+
+As of Vault 1.9, the Vault Audit Log will contain a `client_id` field in the request. The `client_id` will
+be empty for requests that are untracked by the Audit Log. Such requests are unauthenticated requests / 
+login requests, requests made with the root token, and wrapped requests. The `client_id` field can also 
+contain an Entity ID for requests that are made with tokens with entities, or a unique client ID for 
+non-entitiy tokens. 
+
+Consumers of the audit log will be able to distinguish between a non-entity `client_id` and a 
+`client_id` that is also an entity IDs by either:
+A. Comparing the value of the `client_id` in the request with the value of the `entity_id` in the `Auth`
+section of the response and checking if they are the same.
+B. Checking that the format of the `client_id` is `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`(this signifies 
+that the `client_id` is an `entity_id`, versus a `client_id` which appears to be a base64 encoding 
```

Comment:
```suggestion
that the `client_id` is an `entity_id`. A `client_id` which appears to be a base64 encoding 
```

Reply:
this suggestion is a bit strange since the stuff is in parens, but it didn't read right to me as it was

Reply:
Definitely! Removed the parens to make this more clear.